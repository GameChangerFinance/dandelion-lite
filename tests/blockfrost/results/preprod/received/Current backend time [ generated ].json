{
  "server_time": 1748318701942
}
