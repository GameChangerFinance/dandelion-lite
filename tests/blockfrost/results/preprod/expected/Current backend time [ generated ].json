{
  "server_time": 1748318705162
}
